app/components/widgets/CodeEditor.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <div class="toolbar">
-      <i class="icon-reset" v-tooltip="$t('Restore Defaults')" @click="restoreDefaults" />
+      <!-- <i class="icon-reset" v-tooltip="$t('Restore Defaults')" @click="restoreDefaults" /> -->
```

Comment:
How to restore defaults now?

Reply:
we never correctly restored defaults in the first place; this button has been broken since before the redesign